packages/upload-api/src/blob/add.js

```diff
@@ -161,6 +161,11 @@ async function allocate({ context, blob, space, cause }) {
 
   // 3. if not already allocated (or expired) execute `blob/allocate`
   if (!blobAllocateReceipt) {
+    // Create allocation task and save it
+    const saveTask = await context.tasksStorage.put(task)
+    if (!saveTask.ok) {
+      return saveTask
+    }
```

Comment:
`ucan/conclude` seems to assume that `blob/allocate` task is stored here, but nothing was storing it which caused errors when `http/put` receipts were submitted. I'm not sure what @vasco-santos had intended here, as some tests seemed to manually store these. But then some handlers seemed to save task from the handler.

I think it is reasonable to save task from the handler especially if we intend to look those up later.